app/.idea/libraries/Gradle__com_google_android_gms_play_services_base_17_0_0_aar.xml

```diff
@@ -0,0 +1,12 @@
+<component name="libraryTable">
+  <library name="Gradle: com.google.android.gms:play-services-base:17.0.0@aar">
+    <CLASSES>
+      <root url="jar://$USER_HOME$/.gradle/caches/transforms-2/files-2.1/59575653ac69a5f74f1243f891eb83df/play-services-base-17.0.0/jars/classes.jar!/" />
+      <root url="file://$USER_HOME$/.gradle/caches/transforms-2/files-2.1/59575653ac69a5f74f1243f891eb83df/play-services-base-17.0.0/res" />
+    </CLASSES>
+    <JAVADOC>
+      <root url="jar://$USER_HOME$/.gradle/caches/modules-2/files-2.1/com.google.android.gms/play-services-base/17.0.0/d2c4415b10482b4829dd3708f5391f637ed5816/play-services-base-17.0.0-javadoc.jar!/" />
+    </JAVADOC>
+    <SOURCES />
+  </library>
+</component>
```

Comment:
Is there a reason why this file is in the PR? Same with the other .idea/caches and .idea/libraries files in your PR. I checked the .gitignore and it looks like it should be covered by the .gitignore.
![image](https://user-images.githubusercontent.com/11599574/68364857-6bdaba80-00ec-11ea-8275-16852396cedf.png)